packages/manager/apps/pci-project/src/data/api/pci-project.ts

```diff
@@ -0,0 +1,18 @@
+import { apiClient } from '@ovh-ux/manager-core-api';
```

Comment:
Generally we don't put `pci-` in the file name, `project.ts` is enough and uses the same pattern as the other api files. (The generator generates with `pci-` but we never did that way)